helpers.ts

```diff
@@ -52,6 +52,17 @@ export function trackDownloadProgress(destinationStream: NodeJS.WritableStream,
 	return progressStream;
 }
 
+export function isBuildFromCLI(item: string) {
+	return item.match(/.*\.aar/) ||
+			item.match(".*include.gradle");
```

Comment:
this regex will match strings like:
* filename.include.gradle
* include.gradle
* include.gradle.backup

Is this the desired behavior?

Reply:
yes